wca-regulations.md

```diff
@@ -405,7 +405,9 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - A6b2) If the timer stops before the competitor has solved the puzzle and displays a time of 0.06 seconds or higher, then the attempt is disqualified (DNF). Exception: if the competitor can demonstrate that the timer malfunctioned, they may receive an extra attempt, at the discretion of the WCA Delegate.
     - A6c) The competitor must fully release the puzzle before stopping the timer. Penalty: disqualification of the attempt (DNF). Exception: If no moves have been applied after stopping the timer and before releasing the puzzle, a time penalty (+2 seconds) may be assigned instead, at the discretion of the judge.
     - A6d) The competitor must stop the timer using both hands, placed flat on the sensors with palms down. Penalty: time penalty (+2 seconds).
-    - A6e) After releasing the puzzle, the competitor must not touch or move the puzzle until the judge has inspected the puzzle. Penalty: disqualification of the attempt (DNF). Exception: If no moves have been applied, a time penalty (+2 seconds) may be assigned instead, at the discretion of the judge.
+    - A6e) After stopping the timer, the competitor must not touch or move the puzzle until the attempt has ended (see [Regulation A7c](regulations:regulation:A7c)) or after the judge has explicitly allowed them to do so.
+            - A6e1) Penalty for moving the puzzle after stopping the timer: disqualification of the attempt (DNF).
```

Comment:
I am certain that _moving the puzzle_ is a valid synonym for _applying moves to the puzzle_.